Tests/System.Security.CryptographyTests/HmacSha256Tests.cs

```diff
@@ -0,0 +1,88 @@
+        };
+
+        static ArrayList expectedHashes = new ArrayList()
+        {
+            new byte[] { 0xb0, 0x34, 0x4c, 0x61, 0xd8, 0xdb, 0x38, 0x53, 0x5c, 0xa8, 0xaf, 0xce, 0xaf, 0x0b, 0xf1, 0x2b, 0x88, 0x1d, 0xc2, 0x00, 0xc9, 0x83, 0x3d, 0xa7, 0x26, 0xe9, 0x37, 0x6c, 0x2e, 0x32, 0xcf, 0xf7 },
+            new byte[] { 0x5b, 0xdc, 0xc1, 0x46, 0xbf, 0x60, 0x75, 0x4e, 0x6a, 0x04, 0x24, 0x26, 0x08, 0x95, 0x75, 0xc7, 0x5a, 0x00, 0x3f, 0x08, 0x9d, 0x27, 0x39, 0x83, 0x9d, 0xec, 0x58, 0xb9, 0x64, 0xec, 0x38, 0x43 },
+            new byte[] { 0x77, 0x3e, 0xa9, 0x1e, 0x36, 0x80, 0x0e, 0x46, 0x85, 0x4d, 0xb8, 0xeb, 0xd0, 0x91, 0x81, 0xa7, 0x29, 0x59, 0x09, 0x8b, 0x3e, 0xf8, 0xc1, 0x22, 0xd9, 0x63, 0x55, 0x14, 0xce, 0xd5, 0x65, 0xfe },
+            new byte[] { 0x82, 0x55, 0x8a, 0x38, 0x9a, 0x44, 0x3c, 0x0e, 0xa4, 0xcc, 0x81, 0x98, 0x99, 0xf2, 0x08, 0x3a, 0x85, 0xf0, 0xfa, 0xa3, 0xe5, 0x78, 0xf8, 0x07, 0x7a, 0x2e, 0x3f, 0xf4, 0x67, 0x29, 0x66, 0x5b },
+            new byte[] { 0x60, 0xe4, 0x31, 0x59, 0x1e, 0xe0, 0xb6, 0x7f, 0x0d, 0x8a, 0x26, 0xaa, 0xcb, 0xf5, 0xb7, 0x7f, 0x8e, 0x0b, 0xc6, 0x21, 0x37, 0x28, 0xc5, 0x14, 0x05, 0x46, 0x04, 0x0f, 0x0e, 0xe3, 0x7f, 0x54 },
+        };
+
+        [DataRow("RFC4231 Test case 1", 0, 0, 0)]
+        [DataRow("RFC4231 Test case 2", 1, 1, 1)]
+        [DataRow("RFC4231 Test case 3", 2, 2, 2)]
+        [DataRow("RFC4231 Test case 4", 3, 3, 3)]
+        [DataRow("RFC4231 Test case 6", 4, 4, 4)]
+        [TestMethod]
+        public void TestHMACSHA256Static(
+            string testCase,
+            int keyIndex,
+            int sourceIndex,
+            int hashIndex)
+        {
+            OutputHelper.WriteLine($"Test Case: {testCase}");
+
+            var hash = HMACSHA256.HashData(
+                (byte[])keys[keyIndex],
+                (byte[])sources[sourceIndex]);
+
+            CollectionAssert.AreEqual(
+                (byte[])expectedHashes[hashIndex],
+                hash);
+        }
+
+        [DataRow("RFC4231 Test case 1", 0, 0, 0)]
+        [DataRow("RFC4231 Test case 2", 1, 1, 1)]
+        [DataRow("RFC4231 Test case 3", 2, 2, 2)]
+        [DataRow("RFC4231 Test case 4", 3, 3, 3)]
+        [DataRow("RFC4231 Test case 6", 4, 4, 4)]
+        [TestMethod]
+        public void TestHMACSHA256Compute(
+            string testCase,
+            int keyIndex,
+            int sourceIndex,
+            int hashIndex)
+        {
+            OutputHelper.WriteLine($"Test Case: {testCase}");
+
+            HMACSHA256 hasher = new HMACSHA256((byte[])keys[keyIndex]);
+            hasher.ComputeHash((byte[])sources[sourceIndex]);
+
+            //CollectionAssert.AreEqual(
```

Comment:
why is this commented?

Reply:
Because I was testing something and forgot about this... 😅